<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDlg</class>
 <widget class="QDialog" name="LoginDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refreshBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend1">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>117</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>friend1</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>157</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>friend2</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>197</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>friend3</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>237</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>friend4</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>277</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>friend5</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>197</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>tag3</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>237</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>tag4</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>157</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>tag2</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag1">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>117</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>tag1</string>
   </property>
  </widget>
  <widget class="QLabel" name="tag5">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>277</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>tag5</string>
   </property>
  </widget>
  <widget class="QLabel" name="name5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>267</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>eee</string>
   </property>
  </widget>
  <widget class="QLabel" name="name3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>187</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>ue</string>
   </property>
  </widget>
  <widget class="QLabel" name="name4">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>227</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>ty</string>
   </property>
  </widget>
  <widget class="QLabel" name="name2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>147</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Misa</string>
   </property>
  </widget>
  <widget class="QLabel" name="name1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>107</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Meiko</string>
   </property>
  </widget>
  <widget class="QLabel" name="friendlist">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>67</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>FriendList</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag5">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>280</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>问题5：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag3">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>问题3：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag4">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>240</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>问题4：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>问题2：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>120</y>
     <width>41</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>问题1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag1_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>如何让大脑保持高速运行？</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag1_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>想学的东西太多时间有限怎么办？</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag1_4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>200</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>有哪些好的网课网站？</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag1_5">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>240</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>有哪一本书是你读了三遍以上的？</string>
   </property>
  </widget>
  <widget class="QLabel" name="Qtag1_6">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>280</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>有哪些让人欲罢不能的学习方法？</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ad">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>551</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255, 255, 255, 0);
background:transparent;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255);
font: italic 18pt &quot;Marker Felt&quot;;</string>
   </property>
   <property name="text">
    <string>好友列表</string>
   </property>
  </widget>
  <widget class="QWidget" name="adinf" native="true">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>751</width>
     <height>501</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0)</string>
   </property>
   <widget class="QPushButton" name="returnf">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>410</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>691</width>
      <height>361</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>在移动互联网、大数据、超级计算、传感网、脑科学等新理论新技术以及经济社会发展强烈需求的共同驱动下，人工智能加速发展。大数据驱动知识学习、跨媒体协同处理、人机协同增强智能、群体集成智能、自主智能系统成为人工智能的发展重点，人工智能发展进入新阶段。“2019第四届中国（上海）国际人工智能展览会暨人工智能产业大会”是维科网OFweek的品牌展览会，于2019年8月9日-11日在上海新国际博览中心举办。本届展会由中国高科技行业OFweek维科网和高科会主办，OFweek人工智能网承办，以“智慧科技·能创未来”为主题，为世界人工智能行业发展创建了一个技术交流、贸易合作、提高企业品牌的平台。

　　本届展会展览面积约11500平方米，展期3天，预计将吸引500家人工智能优秀企业和来自世界各地的人工智能行业内50000名专业观众。主办方对本届展会进行了专业的展区规划及分区，展区包括“核心技术展区”、“智能机器人展区”、“智能终端展区”、“智能制造展区”、“智慧教育展区”、“智慧城市及物联网展区”集中展示人工智能核心技术研发及相关应用领域前沿新品。本次展览会在高效对接展商及观众、整合人工智能全产业链同时，将同全球人工智能优秀企业共同打造“人工智能产业生态圈”，引领世界人工智能向前发展。</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="go">
   <property name="geometry">
    <rect>
     <x>631</x>
     <y>130</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Baskerville&quot;;</string>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_2">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>170</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Baskerville&quot;;</string>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_3">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>210</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Baskerville&quot;;</string>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_4">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>250</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Baskerville&quot;;</string>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="go_5">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>290</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 13pt &quot;Baskerville&quot;;</string>
   </property>
   <property name="text">
    <string>查看</string>
   </property>
  </widget>
  <widget class="QPushButton" name="know">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>了解</string>
   </property>
  </widget>
  <widget class="QPushButton" name="know_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>了解</string>
   </property>
  </widget>
  <widget class="QPushButton" name="know_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>了解</string>
   </property>
  </widget>
  <widget class="QPushButton" name="know_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>了解</string>
   </property>
  </widget>
  <widget class="QPushButton" name="know_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>280</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>了解</string>
   </property>
  </widget>
  <widget class="QWidget" name="user" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>251</width>
     <height>381</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QPushButton" name="userreturn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>112</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Return</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_plan">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>复习期末</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>年级</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>专业</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>积分</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>兴趣爱好</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_credits">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>675</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_usename">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Meiko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_grade">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>大二</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 255, 255)</string>
    </property>
    <property name="text">
     <string>短期计划</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_hobby">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>看电影</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_major">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>翻译</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
